src/WingetCreateCLI/Commands/SubmitCommand.cs

```diff
@@ -115,12 +115,24 @@ private async Task<bool> SubmitManifest()
             {
                 Manifests manifests = new Manifests();
                 manifests.SingletonManifest = Serialization.DeserializeFromPath<SingletonManifest>(expandedPath);
+
+                if (this.Replace && !await this.ValidateReplaceArguments(manifests.SingletonManifest.PackageIdentifier, manifests.SingletonManifest.PackageVersion))
+                {
+                    return false;
+                }
```

Comment:
Not a suggestion but more of a note to self: 
We should probably remove this condition and provide proper error handling since we only support submitting multifile manifests.

Reply:
Add a TODO comment